src/components/sidebar/sidebar.jsx

```diff
@@ -40,7 +40,9 @@ const Sidebar = ({ menu, currentSection, wrapperRef }) => {
 
   // Scroll to reveal the active link on mount
   useEffect(() => {
-    const activeLink = sidebarRef.current.querySelector(`.${style.linkActive}`)
+    const activeLink = sidebarRef.current.querySelector(
+      `.${style.linkActive.replace('+', '\\+')}`
```

Comment:
`+` is not a valid classname :angry:

Reply:
Does the active link classname include it?

Reply:
Yup

Reply:
Does it the active link class name have + for a reason, or can it be changed rather than doing this here?

Reply:
It gets generated by Gatsby. It wouldn't actually be a problem as a class name usually except it's not officially a valid character and so the query selector doesn't work. The `+` is a [css selector symbol](https://www.w3schools.com/cssref/css_selectors.asp) and so if it's in the classname then it is interpreting it as that instead so we need to escape it

Reply:
Supposedly there's a way to override the active link classname however I tried it and it did not work, I think because we are passing in a class name manually